doc/manual/src/contributing/hacking.md

```diff
@@ -249,3 +249,22 @@ search/replaced in it for each new build.
 The installer now supports a `--tarball-url-prefix` flag which _may_ have
 solved this need?
 -->
+
+### Checking the manual links
+
+The build checks for broken internal links, but this happens late in the process,
+so `nix build .` is not suitable for iterating. To check the manual incrementally, run:
+
+```console
+make html -j $NIX_BUILD_CORES
+```
+
+When iterating on the makefile, run:
+
+```console
+rm $(git ls-files doc/manual/ -o | grep -F '.md') && rmdir doc/manual/src/command-ref/new-cli && make html -j $NIX_BUILD_CORES
+```
+
+If a broken link occurs in a snippet that was inserted into multiple generated files in different directories, use `@docroot@` to reference the `doc/manual/src` directory.
```

Comment:
I'm not sure what this instruction is trying to tell me. In which case does this happen? I suppose you encountered this during debugging, but I'm lacking the context to follow. Maybe `@docroot@` should be documented in the Makefile itself (saying why it exists and when to use it)?

Reply:
It doesn't have a single place in the Makefile either, so this seems like a central place for it.

Reply:
I've linked this from the makefile in 4917ca17ce70c31ee5c0b16a707bfa5ec647f1e1

Reply:
```suggestion
```

Now that you have the sentence in the new subsection.

Reply:
@roberth just this fix and you can merge yourself. I already approved.